src/Svelte/doc/index.rst

```diff
@@ -60,7 +60,7 @@ code to your ``assets/app.js`` file:
     // assets/app.js
     import { registerSvelteControllerComponents } from '@symfony/ux-svelte';
 
-    registerSvelteControllerComponents(require.context('./svelte/controllers', true, /\\.(j|t)sx?$/));
+    registerSvelteControllerComponents(require.context('./svelte/controllers', true, /\.(ts|js|svelte)$/i));
```

Comment:
Yes, this is a mistake! It should actually match the recipe exactly: https://github.com/symfony/recipes/blob/main/symfony/ux-svelte/2.9/manifest.json

Can you update it to match?